tools/llm/run_vlm.py

```diff
@@ -0,0 +1,387 @@
+"""
+.. _run_vlm:
+
+Running VLM inference with Torch-TensorRT
+==========================================================
+
+This script mirrors the style and structure of *run_llm.py*, illustrating a
+Torch-TensorRT (dynamo backend) workflow for Visual-Language Models (VLMs).
+"""
+
+import argparse
+import copy
+import os
+import sys
+from contextlib import nullcontext
+from typing import Tuple
+
+import requests
+import torch
+import torch_tensorrt
+from PIL import Image
+from torchtrt_ext import register_sdpa
+from transformers import AutoModel, AutoProcessor
+from utils import (
+    generate_mm,
+    generate_mm_with_static_cache,
+    record_stats,
+    time_generate_mm,
+)
+
+# -----------------------------------------------------------------------------#
+# Global configuration
+# -----------------------------------------------------------------------------#
+DEVICE = torch.device("cuda:0")
+
+# Register SDPA as a standalone operator.  Converter & lowering pass are defined
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+import transformers.models.qwen2.modeling_qwen2 as mq  # noqa: E402
+
+mq.ALL_ATTENTION_FUNCTIONS["flash_attention_2"] = mq.ALL_ATTENTION_FUNCTIONS["sdpa"]
```

Comment:
Can you try to modify the model config after it is initialized instead of modifying the entries ? 
```
model.config._attn_implementation = "sdpa"
```